content/blog/harbor-at-fosdem-2022.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Harbor at FOSDEM 22"
+author:
+  name: "Orlin Vasilev"
+description: "Harbor - The Container Registry"
+date: 2022-03-24T12:00:00+04:00
+showPageInfo: true
+---
+
+## Harbor - The Container Registry at FOSDEM 2022!
+
+I'm excited to share that I recently had the opportunity to represent the Harbor community at [FOSDEM 2022](https://fosdem.org/2022/). As a longtime FOSDEM attendee (my 10th-ish year in a row), this was my first time speaking at the conference. It was a dream come time to be accepted to the [Containers devroom track](https://fosdem.org/2022/schedule/track/containers/) and talk with my fellow attendees about the Harbor project.
+​
+As everything during the pandemic this year's FOSDEM was 100% virtual. Next year I hope we can meet each other in person at the venue, but until then below is the link to the recorded talk and slides.
```

Comment:
this needs a space or new line between `Harbor project.As ` its showing up without a space in the preview.

Reply:
Strange, the NL is there :) will add another one